jpa/subsystem/src/main/java/org/jboss/as/jpa/transaction/TransactionUtil.java

```diff
@@ -45,19 +53,29 @@
 
     private static volatile TransactionSynchronizationRegistry transactionSynchronizationRegistry;
     private static volatile TransactionManager transactionManager;
+    private static volatile TransactionListenerRegistry transactionListenerRegistry;
```

Comment:
Why is the container managed persistence context not managed by a service? I thought it was managed by PersistenceUnitServiceImpl?